### PR TITLE
Iow 737 add source code

### PR DESCRIPTION
**Pull request by ee-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/test/resources/testData/discrete_ground_water_aqts.csv [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+15,45646128,"field_visit_identifier","USGS","U.S. Geological Survey","335504116541501","USGS-335504116541501","06","GW","Well","33.91775278000000","-116.9050389000000","North American Datum of 1983","1420","1440",null,null,"1993-04-29 12:00:00","UTC"," 0:00","62610","1993-04-29",null,null,"2026",,"National Geodetic Vertical Datum of 1929","USGS","The reported water-level measurement represents a static level","D","Date is accurate to the Day","R",,"S","GW level, steel tape","1200","Approved for publication -- Processing and review completed.","4116.0"
+16,45646128,"field_visit_identifier","USGS","U.S. Geological Survey","335504116541501","USGS-335504116541501","06","GW","Well","33.91775278000000","-116.9050389000000","North American Datum of 1983","1420","1440",null,null,"1993-06-04 12:00:00","UTC"," 0:00","61055","1993-06-04",null,null,"2070",,"National Geodetic Vertical Datum of 1929","USGS","The reported water-level measurement represents a static level","D","Date is accurate to the Day","R",,"S","GW level, steel tape","1200","Approved for publication -- Processing and review completed.","4117.0"
```
Comment (ee-usgs) [inline src/test/resources/testData/discrete_ground_water_aqts.csv:17]: I removed the "measurement_source_code" & "measurement_source" columns, but the contents of the 'level_accuracy' column is just wrong, so I deleted the contents.  I think the 'level_accuracy_code' contents are incorrect as well...
Our current code never does any logic on those columns, so it really has no impact, its just really misleading to look at as sample data.

File: src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRules.java [hunk 2]
```diff
@@ -100,6 +100,28 @@ public void apply(DiscreteGroundWater domObj) {
...
+		//In all other cases, lev_src_cd is left empty
+		//
+		//Ref:  https://internal.cida.usgs.gov/jira/browse/IOW-737
+		//Complete list of legacy NWISWeb lev_src_cd's (of which we are only mapping to 2 of them) :
+		//  https://help.waterdata.usgs.gov/code/water_level_src_cd_query?fmt=html
+		{
+			String agency = StringUtils.trimWhitespace(domObj.measuringAgencyCode);
+			String srcCode = "";    //Default for empty or null
+
+			if (! StringUtils.isEmpty(agency)) {
```
Comment (orosz-usgs) [inline src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRules.java]: I think StringUtils.isEmpty() is deprecated, StringUtils.hasText() can be used. Avoid the not

**Review by orosz-usgs — APPROVED** [review]
Just the one cleanup

File: src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRules.java (hunk 2 above)
Comment (ee-usgs) [reply src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRules.java]: It doesn't seem to be depricated, but hasText is for sure easier to read, so I changed it throughout.
Thanks!

Comment (duselmann) [reply src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRules.java]: My IDE also says it is depricated. FWIW

**Review by duselmann — APPROVED** [review]

File: src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRules.java [hunk 3]
```diff
@@ -1,16 +1,10 @@
...
-import java.util.Optional;
-
+import com.fasterxml.jackson.core.*;
 import org.springframework.util.StringUtils;
 
-import com.fasterxml.jackson.core.JsonFactory;
-import com.fasterxml.jackson.core.JsonParser;
-import com.fasterxml.jackson.core.JsonToken;
+import java.io.IOException;
+import java.util.*;
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRules.java:7]: Sometimes intellij will contract your imports to use the *, I think for most projects we opt not to do this so it's easier to see exactly which imports we're using.

**Review by ssoper-usgs — APPROVED** [review]

**Pull request merged**